### PR TITLE
US99549-US99550-remove the properties because they are no longer behi…

### PR DESCRIPTION
**Pull request by edwardg8**
…nd ld flags

### CODE REVIEW

File: d2l-image-banner-overlay.html [hunk 1]
```diff
@@ -85,10 +85,7 @@ <h1 id="courseName">[[_courseName]]</h1>
 			button-text="[[localize('undo')]]"
 			has-close-button="true"
 			role="alert">
-			<div id="bannerRemovedAlertText" hidden$="[[homepageManageMenuFlag]]">
-				<span>[[_optBackInStart]]</span><d2l-link href$="[[_courseOfferingInfoLink]]">[[localize('courseOfferingInformation')]]</d2l-link><span>[[_optBackInEnd]]</span>
```
Comment (dlockhart) [inline d2l-image-banner-overlay.html]: Is there an opportunity to kill `_optBackInStart`, `_courseOfferingInfoLink`, `_optBackInEnd` and the `courseOfferingInformation` language terms, or are they still used elsewhere?

File: d2l-image-banner-overlay.html [hunk 2]
```diff
@@ -248,14 +235,6 @@ <h1 id="courseName">[[_courseName]]</h1>
 			_onClearImageScrollThreshold: function() {
 				this.$['image-selector-threshold'].clearTriggers();
 			},
-			_onDepartmentBannerFlag: function(newValue) {
```
Comment (edwardg8) [inline d2l-image-banner-overlay.html]: It turned out to be that we can safely remove this observer, as the default has `visible-on-ancestor` property set, thanks Dave for clarifying this!

File: d2l-image-banner-overlay.html (hunk 1 above)
Comment (edwardg8) [reply d2l-image-banner-overlay.html]: I'll take a look!

Comment (edwardg8) [reply d2l-image-banner-overlay.html]: I have these removed now!

**Pull request merged**